hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/SnapshotDiffHandler.java

```diff
@@ -68,6 +68,12 @@ public class SnapshotDiffHandler extends Handler {
       defaultValue = "false")
   private boolean cancel;
 
+  @CommandLine.Option(names = {"--fnnd", "--force-non-native-diff"},
+      description = "perform diff of snapshot without using" +
+          " native libs (optional)",
+      hidden = true)
+  private boolean forceNonNativeDiff;
```

Comment:
Rename CLI param as well if you feel like it.

Reply:
done